docs/DeveloperGuide.md

```diff
@@ -309,6 +309,20 @@ Priorities: High (must have) - * * *, Medium (nice to have) - * *, Low (unlikely
 
 (For all use cases below, the **System** is `LingoGO!` and the **Actor** is the `user`, unless specified otherwise)
 
+**Use case: List flashcards**
+
+**MSS**
+1. User requests to list out flashcards.
+2. LingoGO! shows a list of flashcards.
+
+   Use case ends.
+
+**Extensions**
+* 1a. The given user input is invalid.
+    * 3a1. Lists all flashcards.
```

Comment:
I think we should specify what kind of invalid input would lead to listing all flashcards (i.e. when user tries to list more flashcards than there are available), since there are different kinds of invalild inputs (e.g. negative numbers should not be accepted)

Also, step number should be 1a1., not 3a1